test/integration/targets/azure_rm_webapp/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/azure
+posix/ci/cloud/group3/azure
+unstable
```

Comment:
why unstable? is the test going to be unstable from the beginning?

Reply:
fixed. copy wrong one